pagopa/pn-templates-engine
# Feat/PN-13389: show second page in AAR RADDalt bilingual also for PG

## Short description
Show second page in ARR RADDalt bilingual also for PG 

## List of changes proposed in this pull request
- Remove conditional `noIta` variable for no Italian language 
- Fix copy title in FR

## How to test
Start microservices pn-template-engine and call API (from Postman collection):
- PUT `/templates-engine-private/v1/templates/notification-aar-radd-alt` with PG recipient and blilingual
- Check that exists second page "delega"
- For language FR (set request header x-language), check that gray box `RETIRER UNE COPIE SUR LE TERRITOIRE...` has not 2 lines
